cypress/features/regression/experimental/decimalInput.feature

```diff
@@ -35,6 +35,6 @@ Feature: Decimal input component
     When I set Decimal input to <label>
     Then Decimal input is not set to <label>
     Examples:
-      | label                     |
-      | mpú¿¡üßä                  |
-      | !@#$%^*()_+=~[];:?{}&"'<> |
```

Comment:
I don't think the issue was about the `label`. I thought that it was about typing the `^` inside the input.

Reply:
Looks like the wording on the Cypress test is incorrect to me here, I think it should be `When I set Decimal input to <input>` etc - this test is actually testing special characters inside the input so Dan's change is correct here

Reply:
Yeah. What @sjohnson-sage said. This test is testing the characters in the input field. It confused me a bit at first when I saw `label`. Maybe it's something we should look to change?

Reply:
I think this will send the carat/backtick. It's not a problem, but we should be aware we're not testing the dead key.